admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -56,7 +57,16 @@ def grid(self):
                  request_method='GET',
                  renderer='../templates/edit.jinja2')
     def view(self):
-        return super().edit()
+        to_render = super().edit()
```

Comment:
this is usually called "response", but do not consider this as blocking for the PR if CI pass.